HardwareDefinitions/avnet_sk_rev2_files_for_SDK_installer/avnet_mt3620_sk_rev2.json

```diff
@@ -0,0 +1,90 @@
+{
+  "Metadata": {
+    "Type": "Azure Sphere Hardware Definition",
+    "Version": 1
+  },
+  "Description": {
+    "Name": "Avnet MT3620 Starter Kit (SK) Rev2",
+    "MainCoreHeaderFileTopContent": [
+      "/* Copyright (c) Microsoft Corporation. All rights reserved.",
+      "   Licensed under the MIT License. */",
```

Comment:
We should also update the license, now that AVNET owns the hardware definitions.

Reply:
I'll make this change for ALL the avnet specific hardware files.  Is that what you're asking?

Reply:
Done